test/image/test_image_info.py

```diff
@@ -1,61 +1,26 @@
 import json
-import os
 from io import BytesIO
+from pathlib import Path
 
 import fpdf
 
-from test.utilities import relative_path_to
+HERE = Path(__file__).resolve().parent
 
 
 class TestImageParsing:
-    def break_down_filename(self, image):
```

Comment:
I'd like to preserve this unused method as it documents the naming conventions of the PNG files in this directory

Reply:
Okay, I changed it back.